pkg/gce-cloud-provider/compute/cloud-disk.go

```diff
@@ -18,15 +18,13 @@ import (
 	"strings"
 
 	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/meta"
-	computealpha "google.golang.org/api/compute/v0.alpha"
 	computebeta "google.golang.org/api/compute/v0.beta"
 	computev1 "google.golang.org/api/compute/v1"
 )
 
 type CloudDisk struct {
```

Comment:
Can we add a comment here like "if you want to add alpha API please refer to this PR" ?

Reply:
I added this to cloud provider interface which is what stores the alpha service. I didn;t think we needed it in both places. What do you think?

```
// CloudProvider only supports GCE v1/beta Disk APIs. See
// https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1524
// for how to add GCE alpha Disk support.
```

Reply:
Oh I missed this, yeah one place is fine.